### PR TITLE
fix error for CPM-15 and CPM-17 dataset, and add two cell detection dataset

### PR DESCRIPTION
**Pull request by Younger330**

### CODE REVIEW

**Comment by Younger330** [comment]
Besides, I've noticed that the gland segmentation dataset does not explicitly mention 'gland segmentation.' I suggest adding it, because it is the different task from cell segmentation.

**Comment by Younger330** [comment]
Hello @maduc7,

I have corrected some errors and added some datasets. Please review them at your convenience.

Thank you :)

File: README.md [hunk 1]
```diff
@@ -76,6 +78,7 @@ Kumar [31] | multiple (8) | H&E | [data](https://drive.google.com/drive/folders/
 LC25000 [54] | multiple (lung, colon) | H&E | [data](https://github.com/tampapath/lung_colon_image_set), [paper](https://arxiv.org/ftp/arxiv/papers/1912/1912.12142.pdf) | 25.000 (5 classes) | images + label | patch (768x768) | classi | 60x | 2019
 Lizard [32] | Colon | H&E | [data](https://warwick.ac.uk/fac/cross_fac/tia/data/lizard/), [paper](https://arxiv.org/pdf/2108.11195.pdf) | 495.179 nuclei | images + instance seg mask | seg | patch | 20x (DigestPath + CRAG + GlaS + PanNuke + CoNSeP + TCGA) | 2021
 LYON19 [33] | Multiple (Breast, Colon, Protate) | IHC | [data](https://lyon19.grand-challenge.org/), [paper](https://www.sciencedirect.com/science/article/abs/pii/S1361841519300829) | Test: 441 ROIs - 171.166 cells | images + corrdinates of cell | cell detection | patch | Pannoramic 250Flash II scanner | 2019
+MBM [94] | brone | H&E | [data](https://github.com/ieee8023/countception), [paper](https://arxiv.org/abs/1703.08710) | 44 patches | images+mask |  cell detection | patch (600x600) | 40x | 2017
```
Comment (maduc7) [inline README.md:81]: I guess it's bone and not brone.

Comment (Younger330) [reply README.md:81]: Yes

**Comment by Younger330** [comment]
@maduc7 sorry for my fault, I have corrected it.

File: README.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ Dataset name | Organs | Staining | Link | Size | Data | Task | WSI/Patch | Other
 --- | --- | --- | --- |--- |--- |--- |--- |--- |---
 ACDC-LungHP [1a], [1b] | Lung | H&E | [data](https://acdc-lunghp.grand-challenge.org/), [paper](https://ieeexplore.ieee.org/document/9265237) | Train: 150, Test: 50 | images + xml | seg + classi | wsi | | 2019
 ACROBAT 2022 [66] | Breast | Multiple (IHC, H&E) | [data](https://acrobat.grand-challenge.org/), [paper](https://openreview.net/pdf?id=5TTocO2VI3r) | Train: 750 train; Valid: 100; Test: 300 | images (1 H&E match to 1-4 IHC) + landmarks | registration | wsi | 40x - Hamamatsu | 2022
+Adipocyte [94] | skin | H&E | [data](https://github.com/ieee8023/countception), [paper](https://arxiv.org/abs/1703.08710) | 150 patches | images+mask |  cell detection | patch (600x600) | 40x | 2017
```
Comment (maduc7) [inline README.md:22]: it is `200 patches` and `120X150`.

File: README.md [hunk 3]
```diff
@@ -30,6 +31,7 @@ BCSS [7] | Breast | H&E | [data](https://bcsegmentation.grand-challenge.org/), [
...
 BRACS [62] | Breast | H&E | [data](https://www.bracs.icar.cnr.it/details/), [paper](https://arxiv.org/pdf/2111.04740.pdf) | 547 wsi, 4539 ROIs, 189 Patients | images + label (6 subtypes tumor + normal) | classi (7) | wsi + patch | 40x - Aperio AT2 | 2021
 BreakHis [8] | Breast |	H&E	| [data](https://web.inf.ufpr.br/vri/databases/breast-cancer-histopathological-database-breakhis/), [paper](https://www.inf.ufpr.br/lesoliveira/download/TBME-00608-2015-R2-preprint.pdf) |	7.909 (2480 benign, 5429 malignant)	| images + binary label + tumor type (8) (multiple magnifications: 40x, 100x, 200x, 400x) |	classi |	Patch (700x460) | 40x, 100x, 200x, 400x | 2016
+BRCA-M2C [95] | breast | H&E | [data](https://drive.usercontent.google.com/download?id=1HvIzsOs5FP9OdlJKAnU_PM6tX4B2Q1rk&export=download&authuser=0), [paper](https://arxiv.org/abs/2110.04886) |  120 patches belonging to 113 patients (from TCGA) | images+point annotation |  multi-class cell detection | patch (500x500) | 20x | 2021
```
Comment (maduc7) [inline README.md:34]: Add [github](https://github.com/TopoXLab/Dataset-BRCA-M2C)
Maybe specify: train: 80, valid: 10, test: 30 patches
Maybe add `around 500x500` as apparently not all images are of the same size.

File: README.md [hunk 4]
```diff
@@ -104,6 +107,7 @@ PatchCamelyon [45a], [45b] | Lymph node |	H&E |	[data](https://patchcamelyon.gra
 PATHVQA [80] | Multiple | Multiple | [data](https://drive.google.com/drive/folders/1G2C2_FUCyYQKCkSeCRRiTTsLDvOAjFj5), [paper](https://arxiv.org/pdf/2003.10286.pdf), [github](https://github.com/UCSD-AI4H/PathVQA) | 32,799 open-ended questions from 4,998 images | image + question + answer | VQA | patch/image | | 2020
 Post-NAT-BRCA [74] | Breast | H&E | [data](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=52758117#52758117bcab02c187174a288dbcbf95d26179e8), [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6124665/) | 96 images from 54 patients | images + clinical info + annotation tumor cellularity and cell labels |  | wsi | 20x - Aperio | 2021
 Prostate Fused-MRI-Pathology [83] | Prostate | H&E | [data](https://www.cancerimagingarchive.net/collection/prostate-fused-mri-pathology/) | 114 images from 16 patients | images + tumor Annotations + mpMRI | | wsi | 20x - Aperio | 2016 
+RINGS | prostate | H&E | [data](https://data.mendeley.com/datasets/h8bdwrtnr5/1) | train: 1000 , test: 500 | images+mask |  gland segmentation and tumor segmentation | patch (1500x1500) | 40x | 2021
```
Comment (maduc7) [inline README.md:110]: Is this the associated paper: https://www.sciencedirect.com/science/article/abs/pii/S0933365721000695#:~:text=A%20new%20image%20segmentation%20method,traditional%20and%20deep%20learning%20techniques.
?
Also maybe add(18'851 glands?

**Comment by maduc7** [comment]
Thanks for your contribution, I left a few lasts comments, once you will address them I will merge the PR.

File: README.md (hunk 2 above)
Comment (Younger330) [reply README.md:22]: sorry for my fault, I have corrected it.

**Pull request merged**